### ISSUE TITLE
Progress bar silence during validation for model.fit()

### ISSUE THREAD

**Issue by wuhy08** [tensorflow/tensorflow#45760]
<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>

**System information**
- TensorFlow version (you are using): 2.3.1
- Are you willing to contribute it (Yes/No): Yes

**Describe the feature and the current behavior/state.**

When using model.fit(), even setting `verbose=2` will still disable progress bar in the validation steps. Only the final progress bar with metric shows up when validation steps are finished. After digging the cause, it turns out that an attribute called `_called_in_fit ` is set to `True`, as seen in the following lines. I am not sure what is the reasoning behind disabling progress bar in validation step. But IMO enabling/disabling progress bar option should be left for user to choose.

https://github.com/tensorflow/tensorflow/blob/d3e40668078c96ef46cc3bd59b91e222852d4214/tensorflow/python/keras/callbacks.py#L1001-L1003

**Will this change the current api? How?**

Potentially adding an argument called `val_verbose` in `model.fit()` to enable/disable the progress bar.

**Who will benefit with this feature?**

Anyone whose val dataset is large and requires long time to run.

**Any Other info.**

**Comment by jvishnuvardhan** [comment]
@wuhy08 Sorry for the late response. As keras code moved to separate repo [keras-team/keras](https://github.com/keras-team/keras/issues), can you please open the same issue there. Thanks!

You could also raise a PR if you want to contribute. Thanks!
